dashboard/app/models/concerns/text_to_speech.rb

```diff
@@ -205,8 +211,15 @@ def tts_authored_hints_texts
   end
 
   def tts_update
-    tts_upload_to_s3(tts_short_instructions_text) if tts_should_update_short_instructions?
+    # if the level is a DSL-defined level, then it might be contained in a
```

Comment:
note this isn't true; non-DSL-defined levels (like FreeResponse) can also be contained